packages/chaire-lib-frontend/src/services/map/events/GlobalMapEvents.ts

```diff
@@ -61,4 +73,46 @@ const globalEventDescriptors: MapEventHandlerDescription[] = [
     { type: 'map', eventName: 'mousemove', handler: onMouseMove }
 ];
 
+const applyAestheticChanges = async (boundsGL: MapboxGL.LngLatBounds, zoom: number): Promise<void> => {
+    const localNonce = (applyAestheticChangesNonce = new Object());
```

Comment:
Here please add the following block (but comment it when testing locally ;-) ). This will hide the feature behind a preference. I'll add the preference editing code in a separate PR

```
if (!Preferences.get('features.map.prettyDisplay', false)) {
   return;
}
```